saltstack/base/salt/prerequisites/packages.sls

```diff
@@ -43,6 +43,9 @@ packages_install:
   {% endif %}
       - deltarpm
       - nvme-cli
+      - autossh
+      - openssl
+      - vim-common
```

Comment:
Is this one really needed?

Reply:
`$(xxd -pu <<< $(echo ${TUNNEL_INITIATOR_ID} |`

vim-commons provides the xxd funtionality which is used to decipher the enciphered text. If the xxd package is natively present, (i believe it is on ubunutu not on centos) then you can skip it. But for consistency, you might want to keep it around.

Reply:
Peter, I will resolve this conversation. Reopen if you still have concerns.

Reply:
Ok, got it, thanks